examples/notebooks/MatchingLensGalaxies.ipynb

```diff
@@ -2182,11 +2192,16 @@
    ],
    "source": [
     "mag_norm_om10 = []\n",
+    "test_sed = Sed()\n",
+    "test_sed.readSED_flambda(os.path.join(str(os.environ['SIMS_SED_LIBRARY_DIR']), sed_name_array[0]))\n",
+    "a, b = test_sed.setupCCM_ab()\n",
     "for i, idx in list(enumerate(keep_rows)):\n",
     "    if i % 10000 == 0:\n",
     "        print(i, idx)\n",
     "    test_sed = Sed()\n",
     "    test_sed.readSED_flambda(os.path.join(str(os.environ['SIMS_SED_LIBRARY_DIR']), sed_name_array[i]))\n",
+    "    test_sed.addDust(a, b, A_v=av_array[i], R_v=rv_array[i])\n",
+    "    test_sed.redshiftSED(redshift_list[i])\n",
```

Comment:
You need to set `dimming=True` when calling `redshiftSED`, otherwise your fluxes will be `1+z` too bright.

This change probably applies to the other notebook, as well